charts/cp-schema-registry/values.yaml

```diff
@@ -55,7 +81,9 @@ resources: {}
   #  memory: 128Mi
 
 ## Custom pod annotations
-podAnnotations: {}
+podAnnotations:
+  cluster-autoscaler.kubernetes.io/safe-to-evict: "true"  # Allow the autoscaler to downscale the cluster.
+  sidecar.istio.io/inject: "false"  # Disable Istio side-car
```

Comment:
you should say **why** we are disabling it.
Example: "not useful" or "causing issue X"

Reply:
I don't really know the reason why anymore, we had issues for sure but I can't remember, I could put not useful though

Reply:
maybe just write "because causing issues" (without more details)